python/tests/test_dim.py

```diff
@@ -5,8 +5,8 @@
 
 
 def test_dim():
-    assert Dim.X == Dim.X
-    assert Dim.X != Dim.Y
+    assert 'x' == 'x'
+    assert 'x' != 'y'
```

Comment:
These tests where added for the transition period, when `Dim` is not removed yet. Keep the `Dim` here and below.